packages/create-catalyst/src/commands/integration.ts

```diff
@@ -0,0 +1,118 @@
+import { Command } from '@commander-js/extra-typings';
+import { exec as execCb } from 'child_process';
+import { outputFileSync, writeJsonSync } from 'fs-extra/esm';
+import * as semver from 'semver';
+import { promisify } from 'util';
+import * as z from 'zod';
+
+const exec = promisify(execCb);
+
+interface Manifest {
+  name: string;
+  dependencies: string[];
+  devDependencies: string[];
+  environmentVariables: string[];
+}
+
+export const integration = new Command('integration')
+  .requiredOption('--integration-name <name>', 'Formatted name of the integration')
+  .requiredOption('--source-branch <branch>', 'The branch containing your integration source code')
+  .action(async (options) => {
+    const manifest: Manifest = {
+      name: options.integrationName,
```

Comment:
🍹 Nice to have: a `description` field/option to allow developers to add some fluff.

Reply:
In a future iteration, we will allow developers to add a link to documentation - maybe we can add description if people ask for it 👍